Sample/SampleDelegate/SampleDelegate/AppDelegate.swift

```diff
@@ -100,62 +100,62 @@ extension AppDelegate: RemoteNotificationCapable {
-        case .None:
+    func didReceive(userNotificationPermissions: UserNotificationPermissionsGranted) {
+        switch userNotificationPermissions {
+        case .none:
             // The customer has denied user (e.g. local and loud remote) notification permissions.
             break
-        case .Requested:
+        case .requested:
             // The customer has given us permission for all requested user notifications.
             break
-        case let .Some(grantedPermissions):
-            print("Customer refused to give us permissions for \(requestedUserNotificationSettings().types.exclusiveOr(grantedPermissions))")
+        case let .some(grantedPermissions):
+            print("Customer refused to give us permissions for \(requestedUserNotificationSettings().types.symmetricDifference(grantedPermissions))")
             break
         }
     }
 }
 
 extension AppDelegate: LocalNotificationCapable {
-    func didReceiveLocalNotification(localNotification: UILocalNotification, notificationOrigin: UserNotificationOrigin) {
+    func didReceive(localNotification: UILocalNotification, origin: UserNotificationOrigin) {
         // Process the notification.
     }
 }
 
 extension AppDelegate: OpenURLCapable {
-    func canOpenLaunchURL(launchURLToOpen: URLToOpen) -> Bool {
+    func canOpen(launchURL: URLToOpen) -> Bool {
         // SampleDelegate can handle all URLs given to us at launch! If some URLs shouldn't open your app, return false here.
         return true
     }
     
-    func handleURLToOpen(urlToOpen: URLToOpen) -> Bool {
+    func handle(urlToOpen: URLToOpen) -> Bool {
         // Show UI that corresponds to the opened URL. If you can't open the URL, return false here.
         return true
     }
 }
 
 extension AppDelegate: ShortcutCapable {
-    func canHandleShortcutItem(shortcutItem: UIApplicationShortcutItem) -> Bool {
+    func canHandle(shortcutItem: UIApplicationShortcutItem) -> Bool {
         // SampleDelegate can handle all shortcut items! If your app needs to invalidate shortcut items that were set by a previous version of the app, return false here.
         return true
     }
     
-    func handleShortcutItem(shortcutItem: UIApplicationShortcutItem, completionHandler: () -> Void) {
+    public func handle(shortcutItem: UIApplicationShortcutItem, completionHandler: @escaping () -> Void) {
         // Process the shortcut, then tell iOS that we've handled the shortcut.
         completionHandler()
     }
 }
 
 extension AppDelegate: UserActivityCapable {
-    func canHandleUserActivity(userActivity: NSUserActivity) -> Bool {
+    func canResume(userActivity: NSUserActivity) -> Bool {
         // SampleDelegate can handle all user activity items!
         return true
     }
     
-    func continueUserActivity(userActivity: NSUserActivity, restorationHandler: ([AnyObject]?) -> Void) -> Bool {
+    public func resume(userActivity: NSUserActivity, restorationHandler: @escaping ([Any]?) -> Void) -> Bool {
```

Comment:
`@escaping` used but `restorationHandler` does not escape scope of `resume()`.
`restorationHandler` is never used. Why is it a parameter?